storage/simulation/cluster.go

```diff
@@ -223,45 +223,90 @@ func (c *Cluster) prepareActions() {
 
 // performActions performs a single action, if required, for each range.
 func (c *Cluster) performActions() {
-	// TODO(Bram): convert this to run a single action per store instead of a
-	// single action per range.
-	for _, rangeID := range c.rangeIDs {
-		r := c.ranges[rangeID]
-		nextAction, rebalance := r.getNextAction()
-		switch nextAction {
-		case storage.AllocatorAdd:
-			newStoreID, err := r.getAllocateTarget()
-			if err != nil {
-				fmt.Fprintf(c.actionWriter, "Error: %s\n", err)
-				continue
+	tw := tabwriter.NewWriter(c.actionWriter, 12, 1, 2, ' ', 0)
+	// Each range can only go through one action per epoch. If there is a
+	// conflict, the store will not perform an operation as this simulates two
+	// transactions trying to change the range descriptor at the same time.
+	usedRanges := make(map[proto.RangeID]struct{})
```

Comment:
It's only use is for output and when I said `easy lookup` I meant in the log since they're right next to each other.

```
12:         Store:0     Range:10    ADD:3000
12:         Store:1000  Range:10    ADD:conflict:0
```